website/docs/r/dnat.html.markdown

```diff
@@ -43,6 +44,7 @@ resource "vcd_dnat" "forIcmp" {
 The following arguments are supported:
 
 * `edge_gateway` - (Required) The name of the edge gateway on which to apply the DNAT
+* `network_name` - (Optional *v2.2+*) The name of the organization network name on which to apply the DNAT. This will be required field in the next major version.
```

Comment:
Redundant: 
"The name of the organization network name" should be "The name of the organization network"

Reply:
Fixed

Reply:
It's fixed on snat file but not on dnat.

Reply:
oh, yes. Fixed